lib/katello/tasks/upgrades/4.0/sync_noarch_content.rake

```diff
@@ -0,0 +1,13 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.0' do
```

Comment:
This should be in the 4.1 namespace since we're in the 4.1 dev cycle (ending in around May 4th!)